File: Modul6/PRAK605-2310817110008-Muhammad_Raihan.py
ordo = int(input())
matriks_A = []
matriks_B = []

print("Matriks A")
for i in range(0, ordo, 1):
    baris = list(map(int, input().split()))
    matriks_A.append(baris)

print("Matriks B")
for i in range(0, ordo, 1):
    baris = list(map(int, input().split()))
    matriks_B.append(baris)

print("Matriks A X B")
for i in range(0, ordo, 1):
    for j in range(0, ordo, 1):
        elemen = 0
        for k in range(0, ordo, 1):
            elemen += matriks_A[i][k]*matriks_B[k][j]
        print("{}".format(elemen),end=' ')
    print("")

File: Modul6/PRAK605-2310817110008-Muhammad_Raihan.c
#include <stdio.h>

void main(){
    int ordo;
    scanf("%d", &ordo);
    int matriks_A[ordo][ordo], matriks_B[ordo][ordo];
    
    printf("Matriks A\n");
    for(int i=0; i<ordo; i++){
        for(int j=0; j<ordo; j++){
            scanf("%d", &matriks_A[i][j]);
        }
    }

    printf("Matriks B\n");
    for(int i=0; i<ordo; i++){
        for(int j=0; j<ordo; j++){
            scanf("%d", &matriks_B[i][j]);
        }
    }

    printf("Matriks A X B\n");
    for(int i=0; i<ordo; i++){
        for(int j=0; j<ordo; j++){
            int elemen=0;
            for(int k=0; k<ordo; k++){
                elemen += matriks_A[i][k]*matriks_B[k][j];
            }
            printf("%d ", elemen);
        }
        printf("\n");
    }
}